examples/LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+Copyright 2011 Sandia Corporation. Under the terms of Contract
```

Comment:
Do we need a txt file as well? @davidozog @kholland-intel

Reply:
Yeah, that was exactly it. I'll fix this mistake.